### PR TITLE
sl-assessment

### PR DESCRIPTION
**Pull request by sivakarthikeyanoptit**
Release- 3.7.0

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sl-assessments.env [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+SHIKSHALOKAM_USER_PROFILE_FETCH_ENDPOINT={{sl_assessments_shikshalokam_user_profile_fetch_endpoint | default("/api/user/v1/read")}}
+AUTHORIZATION={{sl_assessments_authorization}}
+MONGODB_URL={{sl_assessments_mongodb_url}}
+SHIKSHALOKAM_BASE_HOST={{sl_assessments_shikshalokam_base_host}}
+
+#Cloud Storage Configuration
+CLOUD_STORAGE={{sl_assessments_cloud_storage}}
+
+
+# Azure Cloud Configuration
```
Comment (rjshrjndrn) [inline ansible/roles/stack-sunbird/templates/sl-assessments.env:23]: are all three are mandatory or optional?
Create a section called
```
##### Cloud config
# Note1
# Note2
var1
var2
```

File: ansible/roles/stack-sunbird/templates/sl-assessments.env [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+
+# AWS Cloud Configuration
+AWS_ACCESS_KEY_ID={{sl_assessments_aws_access_key_id}}
+AWS_SECRET_ACCESS_KEY={{sl_assessments_aws_secret_access_key}}
+AWS_BUCKET_NAME={{sl_assessments_aws_bucket_name}}
+AWS_BUCKET_ENDPOINT={{sl_assessments_aws_bucket_endpoint}}
+AWS_BUCKET_REGION={{sl_assessments_aws_bucket_region}}
+
+#Current in use database name
```
Comment (rjshrjndrn) [inline ansible/roles/stack-sunbird/templates/sl-assessments.env:41]: What's this DB? is it postgres or cassandra ? How to connect to the db ?

File: ansible/roles/stack-sunbird/templates/sl-assessments.env [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+AWS_BUCKET_REGION={{sl_assessments_aws_bucket_region}}
+
+#Current in use database name   
+DB={{sl_assessments_db}}
+
+# Transfer from database name
+TRANSFER_FROM_DB={{sl_assessments_transer_from_db | default("sl-prod")}}
+INTERNAL_ACCESS_TOKEN={{sl_assessments_internal_access_token}}
+
+# Disable learner service check
```
Comment (rjshrjndrn) [inline ansible/roles/stack-sunbird/templates/sl-assessments.env:48]: what's the use of the flag ?

**Review by rjshrjndrn — APPROVED** [review]

File: ansible/roles/stack-sunbird/templates/sl-assessments.env (hunk 2 above)
Comment (sivakarthikeyanoptit) [reply ansible/roles/stack-sunbird/templates/sl-assessments.env:41]: it is cassandra, but details for this i have to check with developer

Comment (sivakarthikeyanoptit) [reply ansible/roles/stack-sunbird/templates/sl-assessments.env]: i have did the changes rajesh

File: ansible/roles/stack-sunbird/templates/sl-assessments.env (hunk 1 above)
Comment (sivakarthikeyanoptit) [reply ansible/roles/stack-sunbird/templates/sl-assessments.env:24]: changed according to your comment

File: ansible/roles/stack-sunbird/templates/sl-assessments.env (hunk 2 above)
Comment (rjshrjndrn) [reply ansible/roles/stack-sunbird/templates/sl-assessments.env]: If you don't have the complete idea, please don't mark it as resolved.

**Pull request closed without merging**